kinto/__init__.py

```diff
@@ -41,7 +41,9 @@ def main(global_config, **settings):
         kwargs['ignore'] = 'kinto.views.flush'
 
     # Redirect default to the right endpoint
-    config.add_route('default_bucket', '/buckets/default{subpath:.*}')
+    config.add_route('default_bucket_collection',
+                     '/buckets/default/{subpath:.*}')
```

Comment:
Now that I see this, I wonder: it could have been possible to define an optional group starting with / in the regexp ?
ex. ``/bucket/default{subpath:^(/.*)?$}``